scripts/app.js

```diff
@@ -4,26 +4,33 @@ import {
   handleCommentSubmission,
 } from "./movie-home/comments.js";
 import { filterAndSortMovies } from "./movie-home/filters.js";
-import { movies } from "./movie-home/data.js";
+import { fetchMovies } from "./movie-home/data-fetcher.js";
 import { initTimerUI } from "./movie-timer/ui.js";
 import { trackElapsedTime } from "./movie-timer/elapsedTime.js";
 
-createSearchAndSortBar();
+let allMovies = [];
```

Comment:
I don't think you need the variable declared globally. You can just use `const allMovies = await fetchMovies();` on line 14.